### DISCUSSION TITLE
Detecting cursor change event

### DISCUSSION THREAD

**Discussion by neohed** [suren-atoyan/monaco-react#233] [Q&A] [answered]
The onChange only fires when the editor content changes.  Is there a way to detect the cursor location change, either from arrow keys or mouse clicks?

**Comment by suren-atoyan** [comment, marked as answer]
@neohed yes, there is. Please check [this](https://github.com/suren-atoyan/monaco-react/issues/240)

**Comment by neohed** [comment]
Ah, that’s great. Thank you
